bin/heudiconv

```diff
@@ -912,11 +916,12 @@ def convert(items, symlink=True, converter=None,
                     #if not is_bids or converter != 'dcm2niix': ##uses dcm2niix's infofile
                     embed_metadata_from_dicoms(converter, is_bids, item_dicoms,
                                                outname, outname_bids, prov_file,
-                                               scaninfo, tmpdir, with_prov)
+                                               scaninfo, tmpdir, with_prov,
+                                               min_meta)
                 if exists(scaninfo):
                     lgr.info("Post-treating %s file", scaninfo)
                     treat_infofile(scaninfo)
-                os.chmod(outname, 0o0444)
+                os.chmod(outname, 0o0440)
```

Comment:
Thanks for the note! We restrict at the higher level access to the directory and then the rest stays world readable... I wonder if it would make sense to read existing permissions first and then just turn off the write bit